cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1265,3 +1265,28 @@ declare interface WorkerOptions {
     };
   };
 }
+
+declare interface WebSocketStreamOptions {
+  protocols?: string[];
```

Comment:
Maybe an unimportant thing but I presume the protocols can only be from a known enumeration, right?

Reply:
No, protocols can be an arbitrary HTTP token string.